Documentation/schemas/profile-manifest-v1.md

```diff
@@ -0,0 +1,94 @@
+# Profile Manifest - v1
+
+A "profile manifest" is a JSON data structure consumed by torcx and usually provided by an external party (e.g. an user) as a configuration file with `.json` extension.
+It contains an ordered list of images to a be applied on a system.
+
+## Changes in v1
+
+Profile manifest v1 includes two new fields:
+ * `format` to specify the type of archive for this addon (e.g. "tgz")
+ * `remote` to optionally specify a remote providing this addon
```

Comment:
Should remote not be required for v1? If we convert stores to be "pseudo-remotes", e.g. `com.coreos.usr` and `com.coreos.var-store`, then remote could be required and we'd just ship some default local ones.

Reply:
I was not planning to convert *all* stores to remotes, thus not having "pseudo-remotes" entities at all. This to avoid blurring lines and ending up with emergent complexity.
In particular, the unversioned vendor store would get its own *real* manifest for `com.coreos.us`, but all the other static stores would not (so `com.coreos.var` would not exist).

If we want to make this field required (which I'm not opposed to), perhaps we should reserve an identifier to convey "this image has no remote source"? In my mind the optional field was already serving this purpose.